File: latihan3 b/Latihan3 Py/Disk.py
class Disk():
#kelas yang digunakan untuk mengimplementasikan Disk
#atribut Disk
	
	type = ""       #type disk
	capacity = 0    #capacity
	Dprice = 0		#price
	
	def __init__(self, type ="", capacity = 0, Dprice = 0):
	#costructor
		self.type = type
		self.capacity = capacity
		self.Dprice = Dprice
	
	#mengeset nilai type
	def settype(self, type):
		self.type = type
	
	#mengeset nilai capacity
	def setcapacity(self, capacity):
		self.capacity = capacity
	
	#mengeset nilai price
	def setprice(self, Dprice):
		self.Dprice = Dprice
     
	#mengembalikan nilai type
	def gettype(self):
		return self.type
	
	#mengembalikan nilai capacity
	def getcapacity(self):
		return self.capacity

	#mengembalikan nilai price
	def getprice(self):
		return self.Dprice

	
	

File: latihan3 b/Latihan3 PHP/Disk.php
<?php

class Disk
{ 
	/*kelas yang digunakan untuk mengimplementasikan Disk/
	/*atribut Disk*/
		
		private $type = "";		//type
		private $capacity = 0;	//capacity
		private $Dprice = 0;	//Dprice
	
		public function __construct($type ="", $capacity = 0, $Dprice = 0){
		/*costructor*/
		
			$this->type = $type;
			$this->capacity = $capacity;
			$this->Dprice = $Dprice;
		}
	
	/*mengeset nilai type*/
	public function settype($type)
	{
		$this->type = $type;
	}
	
	/*mengeset nilai capacity*/
	public function setcapacity($capacity)
	{
		$this->capacity = $capacity;
	}
	
	/*mengeset nilai Dprice*/
	public function setDprice($Dprice)
	{
		$this->Dprice = $Dprice;
	}
	
	/*mengembalikan nilai type*/
	public function gettype()
	{
		return $this->type;
	}
	
	/*mengembalikan nilai capacity*/
	public function getcapacity()
	{
		return $this->capacity;
	}
	
	/*mengembalikan nilai Dprice*/
	public function getDprice()
	{
		return $this->Dprice;
	}
	
	function __destruct(){
		/*desstructor*/
		}
}
?>